src/store/index.js

```diff
@@ -0,0 +1,22 @@
+import Vue from "vue";
+import Vuex from "vuex";
+
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+  state: {
+    stateMovies: [],
+  },
+  getters: {
+    getMovies: (state) => {
+      return state.stateMovies;
+    },
```

Comment:
name `getMovies` suggests it is a method, but when using getters, they are used as values - you are not calling them as functions, right? for this reason it is good practice to name getters in a way that describes what value they represent e.g. `movies` or `movieList`